WordPress/gutenberg
# Classic Block: Move the convert to blocks option from menu to own toolbar button

## Description

Moves the 'Convert to Blocks' option on in the Classic Block from the dropdown menu on the block toolbar to its own explicit toolbar button.

## Types of changes
 - Removes block toolbar menu options
 - Adds block toolbar button

The reason for this change is to make the option to convert from the Classic block to Blocks more explicit and obvious, but not intrusive. As part of deprecating the Calypso Classic editor we are removing the blocking dialog in Calypso that prompts users to convert to blocks when opening Classic content, so the fact that you can convert classic content to blocks is then hidden under a sub menu. Moving it to a toolbar button will help to expose the possibility of converting to blocks to users, which should be useful in all contexts where the block is used

## How has this been tested?
Manually

## Testing

- Check out PR to local gutenberg dev environment  
- Add a classic block
- Make sure a 'Convert to Classic' button appears in the toolbar and that it works as expected

## Screenshots 

Before:
<img width="614" alt="Screen Shot 2020-07-06 at 4 44 02 PM" src="https://user-images.githubusercontent.com/3629020/86556614-ed4e5980-bfa7-11ea-9fad-7a828a92d739.png">

After: 
<img width="581" alt="Screen Shot 2020-07-06 at 4 42 11 PM" src="https://user-images.githubusercontent.com/3629020/86556520-af513580-bfa7-11ea-961b-7d4e0691857c.png">

## Checklist:
